projects/onto-search/src/lib/onto-search-facet/range-facet/models/histogram-model.ts

```diff
@@ -7,9 +7,13 @@ export interface HistogramModel {
   // histogram element scale size
   // element count will be presented in pixels = count * scale
   scale?: number;
+  // The y-coordinate of the center of the circle. The x-coordinate of the center of the circle is the facet label value,
+  // relative to the length of the histogram (runtime calculated)
   baseYPos?: number;
+  // The starting angle, in radians (0 is at the 3 o'clock position of the arc's circle)
+  startAngle?: number;
```

Comment:
it's the canvas api. I'll not limit the dev to draw whatever he wants.

Reply:
Why not expose the whole canvas then?
User should be limited to things that will not break anything and the component would still look good.
If I set here 1 (because I have no idea what a radian is and what this start angle should be) and then spend hours working out why am I seeing arcs and the demo has circles. I think this should be encapsulated.

Reply:
removed